styles/camunda-cloud-navigatedViewer.css

```
@@ -0,0 +1 @@
+@import './base-navigatedViewer.css';
```

In general, we could consider constantly using kebab case.

```suggestion
@import './base-navigated-viewer.css';
```